### PR TITLE
Adjust vw qgep overflow

### PR DESCRIPTION
**Pull request by cymed**
As the wastewater node geometries are 3D, the Linestrings should be too. otherwise I get the error 
```
ERROR:  Geometry has Z dimension but column does not 
SQL state: 22023
```

### CODE REVIEW

**Comment by sjib** [comment]
@cymed Can you add a delta also for this change in the symbology functions - see https://github.com/QGEP/datamodel/tree/master/delta for examples

**Comment by sjib** [comment]
I can run this manually only, if I delete the on_reach_update trigger first.
Would we have to include a delete and re-create of all triggers also or are they anyway included with the views?

See https://github.com/QGEP/QGEP/issues/798 and Pullrequest https://github.com/QGEP/datamodel/pull/146

I think we would have a least to run drop_symbology triggers first also with delete views?
https://github.com/QGEP/datamodel/blob/6aa60794729f24edcd9a7c6d7ad124a605dec4e9/06_symbology_functions.sql#L648

**Comment by ponceta** [comment]
Here we need add the modification to the symbology function to the delta and remove the view and driggers from it since it should be already taken care of through the view recreating.

My only interrogation is the geometry type of this vw_qgep_overflow? If it is a LinestringZ the definition has to be adapted.

**Comment by cymed** [comment]
> Here we need add the modification to the symbology function to the delta and remove the view and triggers from it since it should be already taken care of through the view recreating.
> 
> My only interrogation is the geometry type of this vw_qgep_overflow? If it is a LinestringZ the definition has to be adapted.
> 
> https://github.com/QGEP/datamodel/pull/187/files#diff-6d148c192cfb1fae5eabe0e90fae5614c09faca6803ff6982efd6d401efa9937L6-R6

The idea of the view is to use it inside QGIS (in a later update of the .qgs) to create overflows. Overflows are pseudo-linear, but for visualisations the use of a point is preferable:

- Easier to visualise on large scales
- We can't update the geometry of the Linestring in QGIS, as the geometry is calculated by the view

**Comment by urskaufmann** [comment]
I like this view. If the project is configured, it's easy to work with it.
1. Symbol Regenüberlauf is not from overflow, but from the wastewater-structure (function Regenüberlauf).
2. With vw_qgep_overflow, I find the overflows and it visualises from where to where the overflow is. Normally you can see the line only in a detail scale.
3. This ist much easier than to search in a tab of the wastewater-node, if there exists an overflow
4. The create a new overflow, it is very easy to create the new record, then choose the node and the to-node on the map and that's it (without first opening the form-window of the wastewater-structure)

**Comment by cymed** [comment]
@ponceta the delta from #186 is more recent than this one. We only need to change the vw_oo_overflow.yaml

**Comment by ponceta** [comment]
This requires a subsequent .qgs project adaptation.

File: view/vw_oo_overflow.yaml [hunk 1]
```diff
@@ -3,10 +3,9 @@ table: qgep_od.overflow
 view_name: vw_qgep_overflow
 
 additional_columns:
-  geometry: ST_MakeLine(n1.situation_geometry, n2.situation_geometry)::geometry('LineString',%(SRID)s)
+  geometry: n1.situation_geometry::geometry('PointZ',%(SRID)s)
```
Comment (sjib) [inline view/vw_oo_overflow.yaml:6]: This is now a point instead of a line - I am not sure if that is what @urskaufmann intended this to be.

Comment (sjib) [reply view/vw_oo_overflow.yaml:6]: @cymed can you check with @urskaufmann if this still is what both of you expect of this?

Comment (cymed) [reply view/vw_oo_overflow.yaml:6]: @sjib: @urskaufmann and I agreed to changing the geometry from a line to a point in November. I can forward you the discussions by mail.

Comment (sjib) [reply view/vw_oo_overflow.yaml:6]: OK, then everything is ok

**Pull request merged**